<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestSampleChart</class>
 <widget class="QWidget" name="TestSampleChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestSampleChart</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="SimpleChart" name="chartwidget" native="true"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_btn">
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_data">
       <property name="text">
        <string>random data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="get_data">
       <property name="text">
        <string>get data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_data">
       <property name="text">
        <string>load data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimpleChart</class>
   <extends>QWidget</extends>
   <header>simplechart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
